.snippets/code/basic-substrate/built-in-pallet-configuration.rs

```diff
@@ -0,0 +1,54 @@
+parameter_types! {
+    // The amount of funds that must be reserved for an asset
+    pub const AssetDeposit: Balance = 100;
+    // The amount of funds that must be reserved when creating 
+    // a new transfer approval
+    pub const ApprovalDeposit: Balance = 1;
+    // The basic amount of funds that must be reserved when adding metadata 
+    // to your asset
+    pub const MetadataDepositBase: Balance = 10;
+    // The additional funds that must be reserved for the number of bytes 
+    // you store in your metadata
+    pub const MetadataDepositPerByte: Balance = 1;
+
+    // Maximum lenght for the asset symbol and friendly name
+    pub const StringLimit: u32 = 50;
+}
+
+// Implementing the Assets config trait for the runtime
+impl pallet_assets::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+
+    // Stores the balances in an unsigned integer of 128bits
+    type Balance = u128;
+    // The id of an asset can be defined as an unsigned integer of 64 bits
+    type AssetId = u64;
+    // Uses module Balances as mechanism for currency operations
+    type Currency = Balances;
+
+    // Configure the module by referencing the previously
+    // defined constants
+
+    type AssetDeposit = AssetDeposit;
+    type MetadataDepositBase = MetadataDepositBase;
+    type MetadataDepositPerByte = MetadataDepositPerByte;
+    type ApprovalDeposit = ApprovalDeposit;
+    type StringLimit = StringLimit;
+
+    // More configuration
+
```

Comment:
this line can be deleted

Reply:
I left a line before `// More configuration` to separate what was explicitly shown in the article and what wasn't